app/components/works/detail_component.rb

```diff
@@ -62,7 +62,7 @@ def access
 
     sig { returns(String) }
     def license
-      License::ID_LABEL_HASH[work.license]
+      License.label_for(work.license)
```

Comment:
This is an internal refactoring to use the public interface of the License model, preferring message-sending to using constants. Has no effect outside the component.